x/virtualgroup/keeper/msg_server.go

```diff
@@ -459,40 +480,152 @@ func (k msgServer) StorageProviderExit(goCtx context.Context, msg *types.MsgStor
 func (k msgServer) CompleteStorageProviderExit(goCtx context.Context, msg *types.MsgCompleteStorageProviderExit) (*types.MsgCompleteStorageProviderExitResponse, error) {
 	ctx := sdk.UnwrapSDKContext(goCtx)
 
-	operatorAddr := sdk.MustAccAddressFromHex(msg.StorageProvider)
+	operatorAddr := sdk.MustAccAddressFromHex(msg.Operator)
```

Comment:
I think anyone can send this tx may be more make sense

Reply:
we might need new msg, due to signer is bind to 'storage provider'..